recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - llvmdev =={{ version }}
     - llvm
     - libxml2  # [win]
-    - zlib     # [win]
+    - zlib     # [not osx]
```

Comment:
```suggestion
    - zlib     # [win]
```

Reply:
Happy to try again, but there [were](https://github.com/conda-forge/lld-feedstock/pull/29#issuecomment-933035120) errors without zlib:
> Missing `libz`:
> 
> ```
> [100%] Linking CXX executable ../../bin/lld
> [...]/bin/ld: warning: libz.so.1, needed by [...]/lib/libLLVM-13.so, not found (try using -rpath or -rpath-link)
> [...]/bin/ld: [...]/lib/libLLVM-13.so: undefined reference to `compress2'
> [...]/bin/ld: [...]/lib/libLLVM-13.so: undefined reference to `crc32'
> [...]/bin/ld: [...]/lib/libLLVM-13.so: undefined reference to `compressBound@ZLIB_1.2.0'
> [...]/bin/ld: [...]/lib/libLLVM-13.so: undefined reference to `uncompress'
> ```